### DISCUSSION TITLE
whoever wants an unblocker look at this

### DISCUSSION THREAD

**Discussion by fnfgo** [3kh0/ext-remover#337] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
[incogproxy.onrender.com](https://ingogproxy.onrender.com)
[](https://holyub.onrender.com/)

**Comment by fnfgo** [comment]
> [incogproxy.onrender.com](https://ingogproxy.onrender.com)

holyub.onrender.com

**Reply by fnfgo** [reply]
> > [incogproxy.onrender.com](https://incogproxy.onrender.com)
> 
> holyub.onrender.com

**Reply by fnfgo** [reply]
i messed up the link sorry

**Comment by Aka-but-nice** [comment]
it don't work

**Reply by fnfgo** [reply]
which one?

**Reply by Aka-but-nice** [reply]
both, the first one is a error and the second, doesn't load

**Reply by fnfgo** [reply]
aight try this one incogprxy.onrender.com 
the holy ub one is for games

**Comment by Aka-but-nice** [comment]
ok this one works

**Reply by fnfgo** [reply]
k

**Comment by Aka-but-nice** [comment]
#348 #347

**Comment by 1coolmatt8083** [comment]
do u guys have anything that bypasses light speed agent

**Reply by Blobby-Boi** [reply]
use dextensify  or a webview or ltmeat flood or ltmeat print or exthanger or microsoft labs or just a random proxy

**Reply by 1coolmatt8083** [reply]
how bout for roblox if i wanted to play on now.gg

**Reply by myschoolisass** [reply]
> use dextensify or a webview or ltmeat flood or ltmeat print or exthanger or microsoft labs or just a random proxy


did you read what he said? just go to https://now.gg/apps/roblox-corporation/5349/roblox.html while unblocking your chromebook, works just fine

**Reply by 1coolmatt8083** [reply]
now.gg is blocked by lightspeed agent

**Reply by 1coolmatt8083** [reply]
i need somthing that works and does not detect the vpn

**Reply by 1coolmatt8083** [reply]
or proxy detection meesage

**Reply by myschoolisass** [reply]
**read what i said**
**block lightspeed and then do it**

**Comment by fnfgo** [comment]
lightspeediscool.it.com

**Comment by IvanVerdugo** [comment]
I have linewize, it automatically blocks anything proxy or VPN wise. Not sure if there would ever be a way to get past this.

**Reply by a1game** [reply]
for windows , full pc version {not Browser extension version} just download a 100ish mb file of text and spam open it in text editor then when the icon on taskbar for linewize disapears close the text files
